src/api/parsers/query.cpp

```diff
@@ -60,6 +60,7 @@ const TypeMap &type_map = {
         {"loop",    typeid(int)},               // TODO(kleisauke): Documentation needed.
         {"delay",   typeid(std::vector<int>)},  // TODO(kleisauke): Documentation needed.
         {"fsol",    typeid(bool)},              // TODO(kleisauke): Documentation needed.
+        {"sat",     typeid(float)},
```

Comment:
If we're going to turn this into a recombination operation, this needs to be changed as well:
```suggestion
        {"recomb",  typeid(std::vector<float>)},
```

And something like this needs to be included within `add_value`:
```cpp
    } else if (key == "recomb") {  // type == typeid(std::vector<float>)
        auto matrix = tokenize<float>(value, ",", 9);

        if (matrix.size() == 9) {
            map.emplace("recomb", matrix);
        }
```